bot.js

```diff
@@ -219,6 +219,14 @@ controller.hears(['(sun)$'],'direct_message,direct_mention,mention,ambient',func
     });
 });
 
+controller.hears(['its missing something','it\'s missing something','something is missing'],'direct_message,direct_mention,mention,ambient',function(bot,message) {
```

Comment:
You'll want to use some OR expressions here to capture each phrase.  See some of the other entries as examples.